test/features/placeholders/placeholders.test.js

```diff
@@ -72,4 +72,27 @@ describe('Placeholders', () => {
     expect(tag.getAttribute('href')).to.equal('/modal/800 12345 6789');
     expect(tag.getAttribute('data-attr')).to.equal('/modal/800 12345 6789');
   });
+
+  it('Replaces geo-specific placeholders when disable-geo-placeholders meta content is "off" or meta tag not defined', async () => {
+    config.locale.contentRoot = '/test/features/placeholders/bg';
+    const regex = /{{(.*?)}}|%7B%7B(.*?)%7D%7D/g;
```

Comment:
fixme: regex like this in a test caught my eye :suspect:  luckily it's unnecessary because it's already default parameter in `replaceText` (107)